### PR TITLE
feat: show latest assertions errors in case of retry + added number of retries when success

### LINKED ISSUE #602: Feature: info with number of retry (when success)
Feature: been able to see the number of retry when there is a success.
On 1.0.1 version, you have the number of retry when it fails otherwise need to use logs to see it.

The main idea is to know how long a task can take in different environnement.
You set a high retry but you want to check quickly if the success in another context, is faster.
Ex:
     • Run configurator (test_suites/configurator/set_db.yml)
            • start_mariadb SUCCESS
            [...]
            • test_mariadb_service SUCCESS after 5 retry
            
or have an info: "number of retry is 5"
or from the executor being able to send this variable to the test suite, with like output: num_retry: retry_value

### PR DESCRIPTION
**Pull request by kilianpaquier**
- Added number of retries made when a teststep is in success: closes #602 
- Show the last list of errors when a teststep is in error

### CODE REVIEW

File: process_testcase.go [hunk 1]
```diff
@@ -412,6 +412,9 @@ func (v *Venom) printTestStepResult(tc *TestCase, ts *TestStepResult, tsIn *Test
 				v.Println(" %s", Gray(StatusSkip))
 			} else {
 				v.Println(" %s", Green(StatusPass))
+				if ts.Retries > 1 {
+					v.Println(" \t\t  %s", Green(fmt.Sprintf("It's a success after %d attempts", ts.Retries)))
+				}
```
Comment (fsamin) [inline process_testcase.go:417]:
```suggestion
				if ts.Retries == 0 {
				        v.Println(" %s", Green(StatusPass))
				} else {
					v.Println(" %s (after %d attempts)", Green(StatusPass),  ts.Retries)
				}
```

**Review by fsamin — CHANGES_REQUESTED** [review]
thanks for your PR
here is a small suggestion

File: process_testcase.go (hunk 1 above)
Comment (kilianpaquier) [reply process_testcase.go]: Hello,
Thank you for your feedback, it's been taken into account

**Comment by yesnault** [comment]
Hi @kilianpaquier Thank you for this PR. Can you rebase from master please?

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#1112.0 ✘  
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ✘

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#1109.0 ■   
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ■

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#1109.0 ■   
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ■

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#20.0 ■   
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ■

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#20.0 ■   
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ■

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#20.1 ✘  
* Build
  * Build ✔ 
  * Unit Tests ✔ 
* Tests
  * Acceptance Tests ✔ 
* Package-Analyze
  * Docker ✘  
  * Sonar ✔

**Review by yesnault — APPROVED** [review]

**Review by fsamin — APPROVED** [review]

**Pull request merged**